wear/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -133,4 +138,25 @@ class MessagingManager @Inject constructor(
             }
         }
     }
+
+    private fun handleDeleteIntent(

```

Comment:
We should move this and the same method from phone to `common` as well as the function is nearly identical here.